tss-esapi/src/utils/mod.rs

```diff
@@ -272,9 +272,20 @@ pub fn create_unrestricted_signing_ecc_public(
         .build()
 }
 
+/// Container for public key values
 #[derive(Debug, Clone, Serialize, Deserialize, Zeroize, PartialEq, Eq)]
 pub enum PublicKey {
+    /// RSA public modulus (see 27.5.3.4 in Architecture spec)
```

Comment:
It is a little bit problematic to refer to the specification in this matter. Because no where are we stating what version of the specification we use. So in my opinion it would be better to quote of the specification instead.

```
/// RSA public modulus
///
/// # Details
/// * Trusted Platform Module Library, Part 1: Architecture, Family “2.0”, Level 00 Revision 01.59*
/// >
/// >
```

I do not know if rust doc supports quotation of the type used in markdown. But the above was just an example of a better way to do it my opinion.

Reply:
Yeah, either way is fine.